wp-content/plugins/core/src/Admin/Admin_Subscriber.php

```diff
@@ -21,29 +21,29 @@ private function editor(): void {
 		$this->editor_formats();
 	}
 
-	private function editor_styles() {
-		add_filter( 'block_editor_settings', function ( $settings ) {
+	private function editor_styles(): void {
+		add_filter( 'block_editor_settings', function ( array $settings ): array {
 			return $this->container->get( Editor_Styles::class )->remove_core_block_editor_styles( $settings );
 		}, 10, 1 );
-		add_action( 'enqueue_block_editor_assets', function () {
+		add_action( 'enqueue_block_editor_assets', function (): void {
 			$this->container->get( Editor_Styles::class )->enqueue_block_editor_styles();
 		}, 10, 0 );
-		add_filter( 'tiny_mce_before_init', function ( $settings ) {
+		add_filter( 'tiny_mce_before_init', function ( array $settings ): array {
 			return $this->container->get( Editor_Styles::class )->mce_editor_body_class( $settings );
 		}, 10, 1 );
-		add_action( 'admin_init', function () {
-			return $this->container->get( Editor_Styles::class )->enqueue_mce_editor_styles();
+		add_action( 'admin_init', function (): void {
+			$this->container->get( Editor_Styles::class )->enqueue_mce_editor_styles();
 		}, 10, 1 );
 	}
 
-	private function editor_formats() {
-		add_filter( 'teeny_mce_buttons', function ( $buttons, $editor_id ) {
+	private function editor_formats(): void {
+		add_filter( 'teeny_mce_buttons', function ( array $buttons, string $editor_id ): array {
```

Comment:
Is there any way to make it ignore the args in the closure, these should not be typed because WordPress does not send reliable types.

It should be like this:

```php
add_filter( 'teeny_mce_buttons', function ( $buttons, $editor_id ): array {
			return $this->container->get(Classic_Editor_Formats::class)->teeny_mce_buttons( (array) $buttons, (string) $editor_id );
		}, 10, 2 );
```

Reply:
we _could_ [globally ignore closure return types](https://psalm.dev/docs/running_psalm/issues/MissingClosureReturnType/) with `MissingClosureReturnType` in the [config suppression](https://psalm.dev/docs/running_psalm/dealing_with_code_issues/#config-suppression), but I'm sure we use them elsewhere we'd like to check types.

Reply:
Ahh that rule is for the return type of the closure, not the parameters being passed into it.